students/hranic/module-2-html/exercises/02-book/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>A book about markup languages</title>
+</head>
+<body>
+    <h1 lang="en">Markup languages</h1>
+
+    <p lang="en">A markup language is a computer language that uses tags to define elements within a document. Markup languages are used to annotate a document in a way that is syntactically distinguishable from the text, and can assign semantic meaning to the text.</p>
+
+    <h2>Types of markup languages</h2>
+
+    <dl>
```

Comment:
Sviđa mi se što je ovdje korišten `<dl>` element! Ali nije korišten ispravno. Kod `<dl>` elementa trebamo imati parove `<dt>` i `<dd>`, odnosno parove "term" i "description". Primjerice:

```
<dl>
  <dt>Presentational markup</dt>
  <dd>Used by traditional word-processing systems...</dd>
</dl>
```

Prihvatio bih i simboličku listu, odnosno `<ul>`.